src/functionalTest/java/uk/gov/hmcts/reform/professionalapi/client/ProfessionalApiClient.java

```diff
@@ -480,6 +480,18 @@ public void retrieveBadRequestForPendingOrganisationWithPbaEmail(String email, S
 
     }
 
+    public Map<String, Object> retrievePbaAccountsForAnOrganisationExternal(HttpStatus status, RequestSpecification requestSpecification) {
+
+        Response response = requestSpecification
+                .get("/refdata/external/v1/organisations")
+                .andReturn();
+

```

Comment:
this method also not required ..deleted from demo branch so delete this .

Reply:
done